fieldservice_vehicle_stock/migrations/12.0.1.2.1/post-migration.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(env, version):
+    if not version:
+            return
+
+    env.execute("""UPDATE stock_picking_type
+                SET require_vehicle_id = True
+                WHERE name in ('Vehicle Loading', 'Vehicle Returns',
+                'Location Pickup', 'Location Delivery');
+    """)
```

Comment:
@osi-scampbell Directory needs to be renamed.

Reply:
@max3903 when you say the directory needs to be renamed you are referring to the 12.0.1.2.1 directory? I thought it was supposed to be named after the previous version so any version after it will cause script to run?